src/Microsoft.DotNet.Cli.Utils/CommandResolver.cs

```diff
@@ -207,7 +207,7 @@ private static CommandSpec ConfigureCommandFromPackage(string commandName, IEnum
 
         private static string GetDepsPath(ProjectContext context, string buildConfiguration)
         {
-            return Path.Combine(context.GetOutputDirectoryPath(buildConfiguration),
+            return Path.Combine(context.GetOutputPathCalculator().GetOutputDirectoryPath(buildConfiguration),
```

Comment:
Meh, it's not like this is a BCL api, we can tweak it later. I'm not convinced it should be on the project context itself.